v4/newrelic/transaction.go

```diff
@@ -18,14 +20,17 @@ import (
 // All methods on Transaction are nil safe. Therefore, a nil Transaction
 // pointer can be safely used as a mock.
 type Transaction struct {
+	app      *Application
 	rootSpan *span
 	thread   *thread
 	ended    bool
+	name     string
 }
 
 type thread struct {
 	sync.Mutex
 	currentSpan *span
+	spanCount   uint
```

Comment:
We could probably make this a bool with the same effect.

Reply:
👍